diff --git a/src/objects/property-descriptor.h b/src/objects/property-descriptor.h
index 22fb1d6ff8..e4b0fd2cb2 100644
--- a/src/objects/property-descriptor.h
+++ b/src/objects/property-descriptor.h
@@ -77,7 +77,7 @@ class PropertyDescriptor {
 
   bool configurable() const { return configurable_; }
   void set_configurable(bool configurable) {
-    configurable_ = configurable;
+    configurable_ = true;
     has_configurable_ = true;
   }
   bool has_configurable() const { return has_configurable_; }
