diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index 99aadfa065fae..d1b47acc2badc 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -1934,17 +1934,19 @@ ChromeContentBrowserClient::GetOriginsRequiringDedicatedProcess() {
 // Sign-in process isolation is not needed on Android, see
 // https://crbug.com/739418.
 #if !defined(OS_ANDROID)
-  isolated_origin_list.push_back(
-      url::Origin::Create(GaiaUrls::GetInstance()->gaia_url()));
-#endif
-
-#if BUILDFLAG(ENABLE_EXTENSIONS)
-  auto origins_from_extensions = ChromeContentBrowserClientExtensionsPart::
-      GetOriginsRequiringDedicatedProcess();
-  std::move(std::begin(origins_from_extensions),
-            std::end(origins_from_extensions),
-            std::back_inserter(isolated_origin_list));
-#endif
+  // accounts.google.com
+  // isolated_origin_list.push_back(
+  //     url::Origin::Create(GaiaUrls::GetInstance()->gaia_url()));
+#endif
+
+// chrome.google.com
+// #if BUILDFLAG(ENABLE_EXTENSIONS)
+//   auto origins_from_extensions = ChromeContentBrowserClientExtensionsPart::
+//       GetOriginsRequiringDedicatedProcess();
+//   std::move(std::begin(origins_from_extensions),
+//             std::end(origins_from_extensions),
+//             std::back_inserter(isolated_origin_list));
+// #endif
 
   // Include additional origins preloaded with specific browser configurations,
   // if any.  For example, this is used on Google Chrome for Android to preload
